tests/integration/dom/scroll-to-test.js

```diff
@@ -0,0 +1,113 @@
+import { module, test } from 'qunit';
+import hbs from 'htmlbars-inline-precompile';
+import {
+  render,
+  scrollTo,
+  setupContext,
+  setupRenderingContext,
+  teardownContext,
+} from '@ember/test-helpers';
+
+module('DOM Helper: scroll-to', function(hooks) {
+  hooks.beforeEach(async function() {
+    await setupContext(this);
+    await setupRenderingContext(this);
+  });
+
+  hooks.afterEach(async function() {
+    await teardownContext(this);
+  });
+
+  test('Scroll in vertical direction', async function(assert) {
+    assert.expect(1);
+
+    let currentScrollPosition = 0;
+    let scrollAmount = 50;
+    this.callback = e => {
+      currentScrollPosition = e.target.scrollTop;
+    };
+
+    await render(hbs`
+      <style>
+         .container {
+           height: 200px;
+           overflow-y: auto;
+         }
+         .item {
+           height: 100px;
+         }
+      </style>
+      <div class="container" onscroll={{action callback}}>
+        <ul>
+        <li class="item">A</li>
+        <li class="item">B</li>
+        <li class="item">C</li>
+        <li class="item">D</li>
+        <li class="item">E</li>
+        </ul>
+      </div>
+      `);
+
+    await scrollTo('.container', 0, scrollAmount);
+
+    assert.equal(
+      currentScrollPosition,
+      scrollAmount,
+      'After use of the `scrollTop` a paint cycle is triggered and the callback is called'
+    );
+  });
+
+  test('Scroll in horizontal direction', async function(assert) {
+    let currentScrollPosition = 0;
+    let scrollAmount = 50;
+    this.callback = e => {
+      currentScrollPosition = e.target.scrollLeft;
+    };
+
+    await render(hbs`
+      <style>
+         .container {
+           width: 200px;
+           overflow-x: auto;
+           white-space: nowrap;
+         }
+         .item {
+           width: 100px;
+           height: 100px;
+           display: inline-block;
+         }
+      </style>
+      <div class="container" onscroll={{action callback}}>
+        <div class="item">A</div>
+        <div class="item">B</div>
+        <div class="item">C</div>
+        <div class="item">D</div>
+        <div class="item">E</div>
+      </div>
+      `);
+
+    await scrollTo('.container', scrollAmount, 0);
+
+    assert.equal(
+      currentScrollPosition,
+      scrollAmount,
+      'After use of the `scrollLeft` a paint cycle is triggered and the callback is called'
+    );
+  });
+
+  test('Scroll on window works', async function(assert) {
+    await render(hbs`
+      <style>
```

Comment:
I'm actually thinking we may not want to support scrolling the `window`

Reply:
Where did we land on this? Seems like we kept around support for scrolling window, right? If so, we need to figure out how to prevent the mutation of global styles (which AFAICT can't be "undone"?).

I'm thinking you could tweak to manually set these styles on `body` and then ensure they are cleaned up (either in a `finally` or `afterEach`).

Reply:
I kept the code, but I'm not sure that its useful functionality for the helper to have. If you dont I'll remove the code and the test

Reply:
Ya, I don't feel strongly and it seems vaguely better to start with the "simpler" thing and add support for window scrolling independently. As long as we assert when `window` is used, we can always add that as a feature in the future.